index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Website</title>
+    <link rel="stylesheet" href="mystyle.css">
+    <link rel="icon" href="./favicon.ico" type="image/x-icon">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+    <header>
+        <nav class="flex-container">
+            <div class="logo-div">My Logo</div>
+            <div class="hamburger_icon">        <img src="img/hamburger-menu-icon.webp" width="20">
+            </div>
+        </nav>
+    </header>
+    <section class="section_class">
+        <h1 class="section_head"> 
+            I’m Mariane Glad to see you!
+        </h1>
+
+        <p class="section_body">
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+
+        <a href="microverse.org" class="connect">LET’S CONNECT</a>
+
+        <ul class="flex-container item_first">
+             <li><a href="microverse.org" class="item"><img src="img/svg/Vector-5.svg"/></a></li>
+             <li><a href="microverse.org"><img src="img/svg/Vector-6.svg"/></a></li>
+             <li><a href="microverse.org"><img src="img/svg/Vector-7.svg"/></a></li>
+             <li><a href="microverse.org" class="item"><img src="img/svg/Vector-8.svg"/></a></li>
+             <li><a href="microverse.org" class="item"><img src="img/svg/Vector-9.svg"/></a></li>
+             <li><a href="microverse.org" class="item"><img src="img/svg/Vector-10.svg"/></a></li>
+        </ul>
```

Comment:
- [optional] Kindly ensure that the social media handles are aligned properly according to the template given. 👍  

## Figma design
![papaa](https://user-images.githubusercontent.com/103386183/205618891-c2b386b3-bdf9-46d0-9071-1683170c5dd6.png)